keyboards/ergodox_ez/matrix.c

```diff
@@ -208,10 +208,6 @@ static matrix_row_t read_cols(uint8_t row) {
       return 0;
     } else {
       uint8_t data    = 0;
-      mcp23018_status = i2c_start(I2C_ADDR_WRITE, ERGODOX_EZ_I2C_TIMEOUT);
```

Comment:
I think we should add a comment here, to explain the dependency with select_row leaving some state which this then depends on. Otherwise, someone might try to change things and wonder why its now broken.